examples/ivr_outbound_dialer.py

```diff
@@ -0,0 +1,374 @@
+#Switchy IVR Outbound Dialer
+#
+#Author: Nenad Corbic <ncorbic@sangoma.com>
+#
+#This application is designed to demonstrate Switchy capabilities
+#and ease of use.  The application will connect to actively running
+#FreeSWITCH or Sangoma NSG application. It will then originate
+#a single call and play introductory IVR message. From then
+#on application will wait for user input via DTMF.
+#
+#IVR Menu
+# 911 - Play file contact system admin 
+# 811 - Play file hello
+# 111 - Cause a hangup
+# If user times out on DTMF a hello playback will be heard
+#
+#All user logic should be defined in IVRCallLogic Class.
+#
+#Variables
+#  self.<variables>       are global in nature.
+#  call.vars.['var_name'] should be used for per call info and state
+#
+#Switchy Documentation
```

Comment:
Not following the api docs refs here. Why not just link to the main docs page and be done with it?